ua-snap/rasdaman-ingest
# Mean annual Diurnal Temperature Variation (aka daily temp range)

This PR includes the preprocessing notebook and ingest recipe needed to create a mean annual diurnal temperature variation (aka daily temperature range) coverage for the Arctic-EDS. Note - the original EDS plate referred to it as the former, diurnal temperature variation, but I use "dtr" (daily temperature range) interchangeably because this matches the equivalent `xclim` index.

The content here was used to create the mean_annual_dtr coverage that exists on Apollo.

To test this PR:

* Run the notebook on Atlas! All the cells should execute in sequence. You'll likely need to tweak a few env vars in the initial setup cell to match your own scratch space structure, and you can change COPY_SOURCE to True to bring the necessary source WRF data over if you don't have a copy.

* Try to run a test copy of the ingest on Apollo from `/tmp` with all the data in a `geotiffs` directory.
